src/XAccountFactory.sol

```diff
@@ -84,8 +83,8 @@ contract XAccountFactory {
         );
     }
 
-    function _setupModule(address module, address proxy, uint256 chainId, address deployer, address port) internal {
-        ISafeMsgportModule(module).setup(proxy, chainId, deployer, port);
+    function _setupModule(address module, address proxy, uint256 chainId, address owner, address port) internal {
+        ISafeMsgportModule(module).setup(proxy, chainId, owner, port);
```

Comment:
Is it possible to add some unit test for this?

Reply:
I think e2e test is enough.